### PR TITLE
Feature/forms practice

### PR DESCRIPTION
**Pull request by majovanilla**
Request code review for the project: 

- [x] Create forms using erb

- [x] Includes new view and controller (/users/new)

- [x]  Includes edit view and controller (/users/id/edit)

- [x] Includes extra credit to show validation errors

Thanks for the review!

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```
Comment (Oghenebrume50) [inline app/controllers/users_controller.rb]: Please delete commented codes

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<h1>Create User Page</h1>
+  
+<%= form_for @user do |form|%>
+    Username: <%= form.text_field :username, value: 'John Smith'%><br>
+    Email: <%= form.text_field :email, value: 'example@example.com'%><br>
+    Password: <%= form.password_field :password%><br>
+    <%= form.submit 'Submit'%>
+<% end %>
```
Comment (Oghenebrume50) [inline app/views/users/new.html.erb]: Use a regular HTML form and form_tag here then comment them out

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### STATUS: Changes Required 

Hi, team good job on this one everything works, just a few things to do to complete this project, see my comment below 

- Create a partial and reuse in the new and edit file

Raphael Noriode

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  validates :username, presence: true
+  validates :email, presence: true
```
Comment (mosaaleb) [inline app/models/user.rb]: - [x] **[OPTIONAL]** We can validate all attributes in one liner if they all share the same validator
`validates :username, :email, :password, presence: true`

**Review by mosaaleb — CHANGES_REQUESTED** [review]
## Hi @majovainilla, @fabricio-garcia :wave:

#### To get this project approved, we need to implement the new html form with three different approaches:

1. `html` forms. :x:
2. rails `form_tag` helper :x:
3. rails `form_for` :heavy_check_mark:

Let's add the first two approaches and comment them out.

- [x] **[OPTIONAL]** Please move out the css from the views. it's not readable to have css styles in your html. The right place is to put it under a `app/assets/stylesheets`.
**Also** there is no need to include the `html`, `head` or `body` tags in your views or partials. It's already in `application.html.erb`. All your partials files and views gets rendered around `<%= yield %>` in `application.html.erb`


#### Please handle the requirements and submit a new code review

##### Muhammad Ebeid

**Review by bolah2009 — APPROVED** [review]
## Looks good to me :+1: 

Good job implementing the suggested changes.
### :heavy_check_mark: Approved

**Pull request merged**